Sources/PackageGraph/Resolution/PubGrub/Term.swift

```diff
@@ -40,6 +40,10 @@ public struct Term: Equatable, Hashable {
         )
     }
 
+    public var supportsPrereleases: Bool {
```

Comment:
obligatory access control nit

```suggestion
    package var supportsPrereleases: Bool {
```

Reply:
✅